### ISSUE TITLE
Clean up the sprite sheet

### ISSUE THREAD

**Issue by bagelbyheart** [bagelbyheart/spacenoider#7]
Currently the sprite sheet is both unorganized AND filled with unused assets. It will be a lot easier to fit those assets in if that's cleared out and rearranged.

**Comment by bagelbyheart** [comment]
Ok, so the first issue I see, disparate numbers of frames of animation. While it's neat that the system can handle it, it would be a lot easier to organize the sheet if there were a set number of frames for different types of entities. Maybe something like:

```
Player   = 8 sprites = 2/row
Enemies  = 4 sprites = 4/row
Weapons  = 4 sprites = 4/row
Elites   = 8 sprites = 2/row
Drops    = 2 sprites = 8/row
UI elems = 2 sprites = 8/row
```

**Comment by bagelbyheart** [comment]
Notes on the original layout (no longer accurate in fix branch)

```
Player currently references {1,2,3}
Driller currently references {17,...,22}
Gunner and Bomber use {33,34,35}

Vulcan uses {4,5,6}
Laser uses {7,8}
Missile uses {9,10}

Bomb UI can be removed (used {38,39})
Health UI uses {54,55}

bomb drop uses {51}
vulcan uses {52}
laser uses {53}
missile uses {56}
health uses {57}
empty drop uses {58}
```

**Comment by bagelbyheart** [comment]
I started setting up the fix to do rows of 8x8 animations when I realized I might want to add 16x16 sprites. I think I'll need to make support for that a separate enhancement request (since it would require making changes to entity generics and draw functions.

So that means

```
row1 - player1, player2
row2 - enemy1, enemy2, enemy3, enemy4
row3 - bullet1, bullet2, bullet3, bullet4
row4 - drop1, drop2, drop3, drop4, ui elements
```

Which still leaves a page of spritesheet for bosses or larger hud elements.
